.github/workflows/actions-compile.yml

```diff
@@ -18,16 +18,34 @@ jobs:
       - name: add missing packages
         uses: ConorMacBride/install-package@v1
         with:
-          apt: libelf-dev libbpf-dev libbsd-dev libsysrepo-dev libcap-dev libmnl-dev libselinux-dev libyang-tools
+          apt: libelf-dev libbpf-dev libbsd-dev libcap-dev libmnl-dev libselinux-dev cmake make gcc libtool-bin
```

Comment:
why adding make and gcc, there are already installed:
  https://github.com/okda-networks/iproute2-sysrepo/actions/runs/7967215693/job/21749657929#step:2:47